app/src/main/res/values/strings.xml

```diff
@@ -26,154 +26,154 @@
     <string name="pref_theme_option_light">Light</string>
-    <string name="pref_prefer_advanced_editor">Prefer Advanced Editor</string>
-    <string name="pref_prefer_advanced_editor_summary">Enable this to directly open Advanced mode to add/edit Servers</string>
-    <string name="pref_sort_server_list">Sort Server List</string>
-    <string name="pref_sort_server_list_summary">Servers are sorted by their name</string>
+    <string name="pref_prefer_advanced_editor">Prefer advanced editor</string>
+    <string name="pref_prefer_advanced_editor_summary">Opens advanced mode when adding/editing servers</string>
+    <string name="pref_sort_server_list">Sorting of servers</string>
+    <string name="pref_sort_server_list_summary">By name</string>
     <string name="pref_gesture">Gestures</string>
-    <string name="pref_single_tap">Single Tap</string>
-    <string name="pref_two_finger_tap">2 Finger Tap</string>
+    <string name="pref_single_tap">Single tap</string>
+    <string name="pref_two_finger_tap">Two-finger Tap</string>
     <string name="pref_double_tap">Double Tap</string>
-    <string name="pref_long_press">Long Press</string>
-    <string name="pref_swipe1">1 Finger Swipe</string>
-    <string name="pref_swipe2">2 Finger Swipe</string>
-    <string name="pref_drag">Drag (Long Press then Swipe)</string>
+    <string name="pref_long_press">Long press</string>
+    <string name="pref_swipe1">1 Finger swipe</string>
+    <string name="pref_swipe2">2 Finger swipe</string>
+    <string name="pref_drag">Drag (long press then swipe)</string>
     <string name="pref_mouse">Mouse</string>
-    <string name="pref_mouse_passthrough">Mouse Passthrough</string>
-    <string name="pref_mouse_passthrough_summary_off">Mouse events are used locally for gestures</string>
-    <string name="pref_mouse_passthrough_summary_on">Mouse events are sent directly to Server</string>
-    <string name="pref_hide_local_cursor">Hide Local (Android) Cursor</string>
+    <string name="pref_mouse_passthrough">Mouse passthrough</string>
+    <string name="pref_mouse_passthrough_summary_off">Using mouse events for gestures on Android</string>
+    <string name="pref_mouse_passthrough_summary_on">Sending all mouse events to server</string>
+    <string name="pref_hide_local_cursor">Hide local (Android) cursor</string>
     <string name="pref_vk">Virtual Keys</string>
     <string name="pref_vk_show_all">Show all keys</string>
-    <string name="pref_vk_open_with_keyboard">Show when Keyboard is open</string>
-    <string name="pref_km">Key Mappings</string>
-    <string name="pref_km_right_alt_to_super">Right Alt to Super</string>
-    <string name="pref_km_language_switch_to_super">Language Switch to Super</string>
+    <string name="pref_vk_open_with_keyboard">Show when keyboard is open</string>
+    <string name="pref_km">Shortcuts</string>
+    <string name="pref_km_right_alt_to_super">Right Alt → Super modifier</string>
```

Comment:
```suggestion
    <string name="pref_km_right_alt_to_super">\"Right Alt\" becomes \"Super\" modifier</string>
```

Reply:
```suggestion
    <string name="pref_km_right_alt_to_super">Right Alt → Super</string>
```
- We don't need `modifier`, Super key can be used on its own.
- Using arrow is more clear than explicit `becomes`

Though, we could use `Super/Windows`. I don't think many people outside Linux world know about Super.